cellxgene_schema_cli/cellxgene_schema/__init__.py

```diff
@@ -0,0 +1,33 @@
+import os
```

Comment:
Yeah, also not entirely sure about this practice. It seems code that goes in __init__.py is usually related to package loading behavior. So maybe move this code to a normal py file, though should work as is.

> In the simplest case, __init__.py can just be an empty file, but it can also execute initialization code for the package or set the __all__ variable, described later

[https://docs.python.org/3/tutorial/modules.html#packages]

@blrnw3 may have stronger opinions

Reply:
I prefer just adding it as a separate method in `validate.py`. Sfaria is using these libraries and I'm afraid we will break with the refactoring